algorithms/ar-bsrh/java/LinearSearch.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
+
+public class LinearSearch {
```

Comment:
Codacy found an issue: [The utility class name 'LinearSearch' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6278826)